vms/components/chain/state.go

```diff
@@ -471,32 +413,25 @@ func (s *State) deduplicate(ctx context.Context, blk snowman.Block) (snowman.Blo
 	s.missingBlocks.Evict(blkID)
 
 	// wrap the returned block and add it to the correct cache
-	return s.addBlockOutsideConsensus(ctx, blk)
+	return s.addBlockOutsideConsensus(blk)
 }
 
 // addBlockOutsideConsensus adds [blk] to the correct cache and returns
 // a wrapped version of [blk]
 // assumes [blk] is a known, non-wrapped block that is not currently
 // in consensus. [blk] could be either decided or a block that has not yet
 // been verified and added to consensus.
-func (s *State) addBlockOutsideConsensus(ctx context.Context, blk snowman.Block) (snowman.Block, error) {
+func (s *State) addBlockOutsideConsensus(blk snowman.Block) (snowman.Block, error) {
```

Comment:
congrats! you've just eliminated all the error path in this method. mind removing the `error` return value ?

Reply:
I thought we had a linter for this 🤔